engine/apps/user_management/migrations/0004_rbac_permissions.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-10-25 11:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user_management', '0003_user_hide_phone_number'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='organization',
+            name='is_rbac_permissions_enabled',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='user',
+            name='permissions',
+            field=models.JSONField(default=list),
```

Comment:
Wondering how expensive could this migration be.

Reply:
hmmm good question. I will add it to my to-do list.

I'll dump the prod `usermanagement_user` table, run the SQL command to add the column, and see how long it takes. Will report back shortly